Cargo.toml

```diff
@@ -40,7 +40,7 @@ env_logger = "0.7.1"
 
 [features]
 default = ["groth16", "multicore"]
-gpu = ["ocl", "itertools", "fs2"]
+gpu = ["ocl", "ff-cl-gen", "fs2"]
```

Comment:
why do we need `ocl` as dependency here, if this is all factored out now?

Reply:
only the field library is factored out. fft, multiexp and elliptic curve ops are still here.